### ISSUE TITLE
wrong test accuracy reported 

### ISSUE THREAD

**Issue by kirk86** [huyvnphan/PyTorch-CIFAR10#2]
Hi and thanks for sharing the weights and the models. Correct me if I'm wrong but I beleive what you report as test accuracy is just the validation accuracy and not the final test accuracy of model evaluated on the untouched test set. You should have `train/validt/est` --> `train_loader/valid_loader/test_loader` but I can only see train and validation loaders and you report test accuracies based on the validation loader if I'm not mistaken?
